integ-test/src/test/java/org/opensearch/sql/legacy/PluginIT.java

```diff
@@ -84,329 +87,355 @@ public void sqlDeleteSettingsTest() throws IOException {
+            + "}";
     actual = updateViaSQLSettingsAPI(settings);
-    expected = new JSONObject("{" +
-        "  \"acknowledged\" : true," +
-        "  \"transient\" : { }," +
-        "  \"persistent\" : {" +
-        "    \"plugins\" : {" +
-        "      \"query\" : {" +
-        "        \"metrics\" : {" +
-        "          \"rolling_interval\" : \"75\"" +
-        "        }" +
-        "      }" +
-        "    }" +
-        "  }" +
-        "}");
+    expected =
+        new JSONObject(
+            "{"
+                + "  \"acknowledged\" : true,"
+                + "  \"transient\" : { },"
+                + "  \"persistent\" : {"
+                + "    \"plugins\" : {"
+                + "      \"query\" : {"
+                + "        \"metrics\" : {"
+                + "          \"rolling_interval\" : \"75\""
+                + "        }"
+                + "      }"
+                + "    }"
+                + "  }"
+                + "}");
     assertTrue(actual.similar(expected));
 
-
     // (3) full expanded form
-    settings = "{" +
-        "  \"persistent\": {" +
-        "    \"plugins\" : {" +
-        "      \"query\" : {" +
-        "        \"metrics\": {" +
-        "          \"rolling_interval\": \"65\"" +
-        "        }" +
-        "      }" +
-        "    }" +
-        "  }" +
-        "}";
+    settings =
+        "{"
+            + "  \"persistent\": {"
+            + "    \"plugins\" : {"
+            + "      \"query\" : {"
+            + "        \"metrics\": {"
+            + "          \"rolling_interval\": \"65\""
+            + "        }"
+            + "      }"
+            + "    }"
+            + "  }"
+            + "}";
     actual = updateViaSQLSettingsAPI(settings);
-    expected = new JSONObject("{" +
-        "  \"acknowledged\" : true," +
-        "  \"transient\" : { }," +
-        "  \"persistent\" : {" +
-        "    \"plugins\" : {" +
-        "      \"query\" : {" +
-        "        \"metrics\" : {" +
-        "          \"rolling_interval\" : \"65\"" +
-        "        }" +
-        "      }" +
-        "    }" +
-        "  }" +
-        "}");
+    expected =
+        new JSONObject(
+            "{"
+                + "  \"acknowledged\" : true,"
+                + "  \"transient\" : { },"
+                + "  \"persistent\" : {"
+                + "    \"plugins\" : {"
+                + "      \"query\" : {"
+                + "        \"metrics\" : {"
+                + "          \"rolling_interval\" : \"65\""
+                + "        }"
+                + "      }"
+                + "    }"
+                + "  }"
+                + "}");
     assertTrue(actual.similar(expected));
   }
 
   /**
-   * Both transient and persistent settings are applied for same settings.
-   * This is similar to _cluster/settings behavior
+   * Both transient and persistent settings are applied for same settings. This is similar to
+   * _cluster/settings behavior
```

Comment:
should we add a `<br>` here to keep the second sentence on it's own line?

Reply:
I think it's fine